tests/installation/start_install.pm

```diff
@@ -134,4 +134,18 @@ sub run {
     }
 }
 
+sub post_fail_hook {
+    my $self = shift;
+    if (check_var('UPGRADE', '1')) {
```

Comment:
This will cause migration test can't get full log, I think you can add y2log upload here and overload the SUPER::post_fail_hook.

Reply:
this is what we need for investigation. before this with default SUPER::post_fail_hook
we can't get anything. see https://openqa.nue.suse.com/tests/7843085#downloads

Reply:
If this is just a workaround, please just upload log needed then  $self->SUPER::post_fail_hook; and add mutex setting to avoid upload log twice, else need investigate why can't upload any load with default post_fail_hook.